### PR TITLE
tech: update color to names and add opacity

### PR DESCRIPTION
**Pull request by sewilee**

### CODE REVIEW

File: client/assets/scss/colors.scss [hunk 1]
```diff
@@ -1,4 +1,59 @@
-$blue: rgba(52, 88, 170, 1);
-$grey: rgba(106, 106, 106, 1);
-$red: rgba(204, 0, 0, 1);
-$white: rgba(255, 255, 255, 1);
\ No newline at end of file
+//blue
+$blue-whale: rgba(15, 47, 63);
+$blue-whale-75: rgba(15, 47, 63, .75);
```
Comment (clee421) [inline client/assets/scss/colors.scss]: Let's not add the ones we aren't using

File: client/assets/scss/colors.scss [hunk 2]
```diff
@@ -1,4 +1,59 @@
...
+$blue-whale-50: rgba(15, 47, 63, .5);
+$blue-whale-25: rgba(15, 47, 63, .25);
+
+$chathams-blue: rgba(48, 87, 110);
```
Comment (clee421) [inline client/assets/scss/colors.scss:3]: What if we're using a blue that's custom and doesn't have a name?

Comment (sewilee) [reply client/assets/scss/colors.scss:3]: they actually all colors have names. you just need to google it, i can always add a link there as a comment to tell whoever if there is a custom color. just input the rgb code and it'll give you a color name.

File: client/assets/scss/colors.scss (hunk 1 above)
Comment (sewilee) [reply client/assets/scss/colors.scss]: i can comment them out and comment them back in when in use or as a note give it a rule to keep it at 75, 50 and 25. no weird ones like .52 opacity

File: client/assets/scss/colors.scss [hunk 3]
```diff
@@ -1,4 +1,32 @@
-$blue: rgba(52, 88, 170, 1);
-$grey: rgba(106, 106, 106, 1);
-$red: rgba(204, 0, 0, 1);
-$white: rgba(255, 255, 255, 1);
\ No newline at end of file
+//Instructions:
```
Comment (clee421) [inline client/assets/scss/colors.scss]: There's no need for instructions. We talked offline about moving these to a `contribution.md` but I don't believe that's required also. All code should go through code review anyway so it should be caught if it's not to standard.

**Review by clee421 — APPROVED** [review]

**Pull request merged**